src/features/Overview/SlotPerformance/TileCard.tsx

```diff
@@ -34,39 +32,17 @@ export default function TileCard({
   metricType,
 }: TileCardProps) {
   const [ref, { width }] = useMeasure<HTMLDivElement>();
-  const selectedSlot = useAtomValue(selectedSlotAtom);
-
-  const tileCountArr = useMemo<unknown[]>(
-    () => new Array(tileCount).fill(0),
-    [tileCount],
-  );
-
-  const liveBusyPerTile = liveIdlePerTile
-    ?.filter((idle) => idle !== -1)
-    .map((idle) => 1 - idle);
-
-  const aggQueryBusyPerTs = queryIdlePerTile
-    ?.map((idlePerTile) => {
-      const filtered = idlePerTile.filter((idle) => idle !== -1);
-      if (!filtered.length) return;
-      return 1 - mean(filtered);
-    })
-    .filter(isDefined);
 
-  const aggQueryBusyPerTile = tileCountArr.map((_, i) => {
-    const queryIdle = queryIdlePerTile
-      ?.map((idlePerTile) => 1 - idlePerTile[i])
-      .filter((b) => b !== undefined && b <= 1);
-
-    if (!queryIdle?.length) return;
-
-    return mean(queryIdle);
-  });
+  const selectedSlot = useAtomValue(selectedSlotAtom);
+  const isLive = !selectedSlot;
```

Comment:
nit: since selectedSlot could technically be 0

```suggestion
  const isLive = selectedSlot === undefined;
```